tools/projmgr/src/ProjMgr.cpp

```diff
@@ -380,6 +380,10 @@ bool ProjMgr::PopulateContexts(void) {
       if (cprojectFile.empty()) {
         ProjMgrLogger::Error(cproject, "cproject file was not found");
         return false;
+      } else if (fs::path(cprojectFile).filename().string() != fs::path(cproject).filename().string()) {
+        ProjMgrLogger::Error(cproject, "cproject filename has case inconsistency");
+        ProjMgrLogger::Info(fs::path(cprojectFile).filename().string(), "should be used instead");
```

Comment:
```suggestion
        ProjMgrLogger::Info(fs::path(cprojectFile).filename().generic_string(), "should be used instead");
```
`generic_string()` prints consistently the same directory separator `/` for all platforms.

Reply:
In this case only the `filename()` part is considered so there should not be any directory separators involved.